.github/workflows/unitpacker-go.yml

```
@@ -0,0 +1,255 @@
+###
+###
+# MAC_APP_CERTIFICATE - Developer ID Application certificate, private key and CA certificate.
+# Export these three items from your Mac's local keystore:
+# * Your (encrypted) private key corresponding to your Developer ID Application certificate
+# * Your Apple Developer ID Application Certificate
+# * Apple's "Developer ID Certification Authority" root certificate.
+# Base64-encode the resulting p12 file and paste the string in this secret.
+# The Application signing identity is used to sign individual files.
+###
+# MAC_APP_CERTIFICATE_PASSPHRASE - passphrase for the MAC_APP_CERTIFICATE private key
+# The passphrase that you used to encrypt your private key during the export.
+###
+# MAC_APP_CERTIFICATE_CN - common name on the MAC_APP_CERTIFICATE
+# This looks something like this: "Developer ID Application: <Your name or company name> (<TeamID>)"
+# If it does not start with "Developer ID Application: " it is the incorrect certificate or name.
+###
+###
+# MAC_INST_CERTIFICATE - Developer ID Installer certificate and private key.
+# Export these two items from your Mac's local keystore:
+# * Your (encrypted) private key corresponding to your Developer ID Installer certificate
+# * Your Apple Developer ID Installer Certificate
+# Base64-encode the resulting p12 file and paste the string in this secret.
+# The Installer signing identity is used to sign packages.
+###
+# MAC_INST_CERTIFICATE_PASSPHRASE - passphrase for the MAC_INST_CERTIFICATE private key
+# The passphrase that you used to encrypt your private key during the export.
+###
+# MAC_INST_CERTIFICATE_CN - common name on the MAC_INST_CERTIFICATE
+# This looks something like this: "Developer ID Installer: <Your name or company name> (<TeamID>)"
+# If it does not start with "Developer ID Installer: " it is the incorrect certificate or name.
+###
+###
+# MAC_STORE_APPLE_ID - Apple ID of the user requesting notarization
+# Usually an e-mail address.
+###
+# MAC_STORE_APP_SPECIFIC_PASSWORD - Application specific password of the user requesting notarization.
+# You can request an app-specific password for your Apple ID user at https://appleid.apple.com
+###
+# MAC_STORE_TEAM_ID - Team ID of the user or organization
+# This is the same 10 characters that are displayed in your certificate CN at the end in brackets.
+###
+# GPG_PRIVATE_KEY - PGP private key block for Linux signing
+# GPG_PASSPHRASE - passphrase for the GPG_PRIVATE_KEY
+##
+
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
+# > Note: An event will not be created when you push more than three tags at once.
+
+name: unitpacker-go
+on:
+  push:
+    tags:
+      - "multisig-v*"
+
+jobs:
+
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      config: ${{ steps.set-config.outputs.config }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: set-config
+        run: |
+          if [ -n "${ACT+x}" ]; then
+            ./compile-unitpacker-config.sh gaiad-v99.0.1
```

Shouldn't this be something like `multisig-v...` and a different script for testing?
```suggestion
            ./compile-unitpacker-config-tools.sh multisig-v99.0.1
```

---

@greg-szabo For this, do we even need worry if github actions is being run locally? The variable "$GITHUB_REF_NAME" is the same for local github actions. So we could remove the if else condition and have it like this

`run: ./compile-unitpacker-config-tools.sh "$GITHUB_REF_NAME"`

---

It's up to you. The condition was put in place for troubleshooting locally. (Running GitHub Actions locally. You don't get a GIT_REF then.) If you're not troubleshooting locally, you don't need it.